### ISSUE TITLE
Consul is not populated on Orc startup

### ISSUE THREAD

**Issue by utdrmac** [openark/orchestrator#758]
This might be a re-open of #497.
Steps:
- Have MySQL Master/Slave cluster running (cluster alias is 'enterprise')
- Have Consul running (In my test, running in -dev mode
- Start Orc (`"ConsulAddress": "127.0.0.1:8500"`)
- Query consul

```
$ consul kv get mysql/master/enterprise
Error! No key exists at: mysql/master/enterprise
```

I had to manually execute `submit-masters-to-kv-stores` in order to see the info in consul

**Comment by shlomi-noach** [comment]
#549 has been merged 17 days ago, which addressed this exact issue. Can you please make sure you have the latest `master`?

**Comment by utdrmac** [comment]
My apologies, I did not pay attention to merge dates. I'm using releases in my training classes cause that is easier for students to install. I'll have to wait for 3.0.14 then.

**Comment by shlomi-noach** [comment]
Which reminds me I need to release... Thank you!

**Comment by utdrmac** [comment]
Resolved by #549
